app/services/get_referees_to_chase.rb

```diff
@@ -0,0 +1,13 @@
+class GetRefereesToChase
```

Comment:
This probably needs to be more explicit too.

Reply:
I think this is fine 👍

Reply:
ok thanks